crates/ruff_linter/src/rules/flake8_simplify/rules/ast_expr.rs

```diff
@@ -216,11 +217,15 @@ fn check_os_environ_subscript(checker: &mut Checker, expr: &Expr) {
         },
         slice.range(),
     );
-    let node = ast::StringLiteral {
-        value: capital_env_var.into_boxed_str(),
-        unicode: env_var.is_unicode(),
-        ..ast::StringLiteral::default()
-    };
+    let node = ast::StringLiteral::default()
+        .with_value(capital_env_var.into_boxed_str())
+        .with_prefix({
+            if env_var.is_unicode() {
+                StringLiteralPrefix::UString
```

Comment:
I dislike `Unicode`, because _all_ strings in Python are unicode... the `u` prefix does nothing at runtime, it's purely there for Python 2 compatibility. (In fact, it was originally removed in Python 3.0, but they added it back in a later Python 3 version because it was so difficult to write code that worked on both Python 2 and Python 3 if the prefix was gone in Python 3.)

Reply:
That's fair. But it is the unicode flag ;) Can you do a quick check on what we call it in other places in the code base? It would be nice if it is consistent.

Reply:
I did a grep -- there's a pyupgrade rule called `UnicodeKindPrefix`, but other than that, I can't see _much_ use of the term `Unicode` across the codebase to describe this prefix...

I'd still rather keep this as-is -- I personally would find it really confusing to refer to these strings as "unicode strings", given that _all_ strings are unicode strings, even without the prefix